codebuild_specs/run_e2e_tests_windows.yml

```diff
@@ -25,9 +25,7 @@ env:
 phases:
   build:
     commands:
-      - export NODE_OPTIONS=--max-old-space-size=8096
-      - export AMPLIFY_DIR=$CODEBUILD_SRC_DIR/out
-      - export AMPLIFY_PATH=$CODEBUILD_SRC_DIR/out/amplify-pkg-linux-x64
-      - echo $AMPLIFY_DIR
-      - echo $AMPLIFY_PATH
-      - source ./shared-scripts.sh && _runE2ETestsWindows
+      - choco install -fy jq
```

Comment:
I will be adding this to the base image at a later date, it only takes a minute to install this dependency.